pkg/environment/environment_test.go

```diff
@@ -15,13 +15,57 @@
 package environment
 
 import (
+	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 )
 
+func TestMustGetKrewPaths_resolvesToHomeDir(t *testing.T) {
+	home := os.Getenv("USERPROFILE")
+	if home == "" {
+		home = os.Getenv("HOME")
+	}
+	if home == "" {
+		t.Fatal("cannot determine HOME or USERPROFILE")
+	}
+	expectedBase := filepath.Join(home, ".krew")
+
+	p := MustGetKrewPaths()
+	if got := p.BasePath(); got != expectedBase {
+		t.Fatalf("MustGetKrewPaths()=%s; expected=%s", got, expectedBase)
+	}
+}
+
+func TestPaths(t *testing.T) {
+	base := filepath.FromSlash("/foo")
+	p := newPaths(base)
+	if got := p.BasePath(); got != base {
+		t.Fatalf("BasePath()=%s; expected=%s", got, base)
+	}
+	if got, expected := p.BinPath(), filepath.FromSlash("/foo/bin"); got != expected {
+		t.Fatalf("BinPath()=%s; expected=%s", got, expected)
+	}
+	if got, expected := p.IndexPath(), filepath.FromSlash("/foo/index"); got != expected {
+		t.Fatalf("IndexPath()=%s; expected=%s", got, expected)
+	}
+	if got, expected := p.InstallPath(), filepath.FromSlash("/foo/store"); got != expected {
+		t.Fatalf("InstallPath()=%s; expected=%s", got, expected)
+	}
+	if got, expected := p.PluginInstallPath("my-plugin"), filepath.FromSlash("/foo/store/my-plugin"); got != expected {
+		t.Fatalf("PluginInstallPath()=%s; expected=%s", got, expected)
+	}
+	if got, expected := p.PluginVersionInstallPath("my-plugin", "v1"), filepath.FromSlash("/foo/store/my-plugin/v1"); got != expected {
+		t.Fatalf("PluginVersionInstallPath()=%s; expected=%s", got, expected)
+	}
+	if got := p.DownloadPath(); !strings.HasSuffix(got, "krew-downloads") {
```

Comment:
What do you test for?

Reply:
DownloadPath() doesn't return some empty string and we can tell the suffix?